### PR TITLE
[LDN-2023] Add breaks and the 'ten years of DODL' talk to the schedule

### PR DESCRIPTION
**Pull request by issyl0**
- As the title says. Need to check the Netlify preview to make sure it looks good though.

### CODE REVIEW

File: data/events/2023-london.yml [hunk 1]
```diff
@@ -163,6 +163,11 @@ program:
     date: 2023-09-21T00:00:00+01:00
     start_time: "10:35"
     end_time: "11:05"
+  - title: "Coffee break"
+    type: custom
+    date: 2023-09-21T23:59:59+01:00
```
Comment (issyl0) [inline data/events/2023-london.yml:168]:
```suggestion
    date: 2023-09-21T00:00:00+01:00
```

File: data/events/2023-london.yml [hunk 2]
```diff
@@ -183,15 +188,40 @@ program:
...
+    date: 2023-09-21T00:00:00+01:00
+    start_time: 15:15
+    end_time: 16:00
+  - title: "Coffee break"
+    type: custom
+    date: 2023-09-21T00:00:00+01:00
+    start_time: 16:00
+    end_time: 16:15
+  - title: "10 Years of DevOpsDays London - Panel Discussion"
+    type: open-space
```
Comment (issyl0) [inline data/events/2023-london.yml:212]:
```suggestion
    type: talk
```

**Comment by chriscoffee** [comment]
> Hmmm, any ideas @chriscoffee how to make the "ten years of devopsdays London" talk title show up without having to add lots of new pages for it? Since we don't know who's "speaking" per se since not everyone has confirmed yet, and we haven't yet figured out what we're actually going to do.
> 
> Screenshot 2023-07-26 at 19 20 42

So couple of things we could do I guess:
- A `type: custom` talk. It doesn't have to link through to anything then until we actually know what it's about.
- You make the page/talk under `devopsdays-london-organizers` and just TBC it like we did with the other speakers who didn't have a talk yet then we alter the speakers later.

**Comment by chriscoffee** [comment]
We could also add a `comment:` https://github.com/devopsdays/devopsdays-web/blob/main/themes/devopsdays-theme/reference.md#program-fields which is a TIL from me given I rarely look at this document 😂

File: data/events/2023-london.yml [hunk 3]
```diff
@@ -183,15 +188,40 @@ program:
...
+    date: 2023-09-21T00:00:00+01:00
+    start_time: 15:15
+    end_time: 16:00
+  - title: "Coffee break"
+    type: custom
+    date: 2023-09-21T00:00:00+01:00
+    start_time: 16:00
+    end_time: 16:15
+  - title: "10 Years of DevOpsDays London - Panel Discussion"
```
Comment (issyl0) [inline data/events/2023-london.yml:211]:
```suggestion
  - title: "10 Years of DevOpsDays London - Discussion"
```

File: data/events/2023-london.yml [hunk 4]
```diff
@@ -183,15 +188,40 @@ program:
...
+    type: talk
```
Comment (issyl0) [inline data/events/2023-london.yml:212]:
```suggestion
    type: custom
```

**Review by issyl0 — COMMENTED** [review]
The ten years talk is a bit undefined as yet, and type: custom probably fits better right now until we know who's actually speaking and why.

**Comment by chriscoffee** [comment]
> The ten years talk is a bit undefined as yet, and type: custom probably fits better right now until we know who's actually speaking and why.

These changes LGTM get someone to :shipit:

**Pull request merged**